src/components/TagCombobox.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <p-combobox v-model="internalValue" placeholder="Search or enter new tag" allow-unknown-value :options="options" :empty-message="emptyMessage">
+    <template #combobox-options-empty>
+      No tags
+    </template>
+    <template #default="scope">
+      <slot v-bind="scope" />
+    </template>
+    <template #option="{ option }">
+      <slot name="option" :option="option" />
+    </template>
+  </p-combobox>
+</template>
+
+<script lang="ts" setup>
```

Comment:
Can we rename this to FlowRunTagCombobox? That way we can also have have tag components for other things.